site-de-reflexion/src/Entity/User.php

```diff
@@ -55,6 +65,23 @@ public function setUsername(string $username): self
         return $this;
     }
 
+    /**
+     * A visual identifier that represents this user.
+     *
+     * @see UserInterface
+     */
+    public function getEmail(): string
+    {
+        return (string) $this->email;
+    }
+
+    public function setEmail(string $email): self
+    {
+        $this->email= $email;
```

Comment:
il fait que tu laisses le $this->username = $email;

sinon tu risques d'avoir une erreur